packages/react-core/src/components/Select/examples/Select.md

```diff
@@ -13,72 +13,113 @@ import BellIcon from '@patternfly/react-icons/dist/esm/icons/bell-icon';
+
+By default, the menu toggle will display a badge to indicate the number of items that a user has selected.
 
 ```ts file="./SelectCheckbox.tsx"
 
 ```
 
 ### Typeahead
+Typeahead is a select variant that replaces the typical button toggle for opening the select menu with a text input and button toggle combo. As a user enters characters into the text input, the select menu will provide suggestions by filtering the select options.
+
+To make a typeahead, pass `variant=typeahead` into the `<MenuToggle>` component and link an `onClick` function to the `<TextInputGroupMain>` component.
 
 ```ts file="./SelectTypeahead.tsx"
 
 ```
 
 ### Typeahead with create option
+If a user enters a value into a typeahead select menu that does not exist, you can allow them to create an option of that value. 
+
+To enable the creation ability, pass `value='create'` into a `<SelectOption>` component. Additionally, you should pass a button with an `onClick` event into the `<TextInputGroupUtilities>` component to display the creation prompt after a user enters a new value. You can use the `placeholder` property to change the default text shown in the text input.
+
+The following example outlines the code implementation required to create a working typeahead menu that allows for creation.
 
 ```ts file="./SelectTypeaheadCreatable.tsx"
 
 ```
 
 ### Multiple typeahead with chips
+A multiple typeahead can be used to allow users to select multiple options from a list. Selected items appear as chips in the select toggle.
+
+When more items than the allowed limit are selected, overflowing items will be hidden under a "more" button. The following example hides items after more than 3 are selected. To show hidden items, select the “more” button. Select "show less" to hide extra items again.
 
 ```ts file="./SelectMultiTypeahead.tsx"
 
 ```
 
 ### Multiple typeahead with create option
 
+When a multiple typeahead offers users a create option, each item that a user creates will be selected. A chip will be placed in the select toggle for each item that is created and selected.
+
 ```ts file="./SelectMultiTypeaheadCreatable.tsx"
 
 ```
 
 ### Multiple typeahead with checkboxes
+By default, a multiple typeahead select allows you to select multiple menu items, placing a checkmark beside selected items. Like basic checkbox select menus, you can add checkboxes to your menu items. This approach may be more accurate and comprehensive for menu scenarios, like filtering.
 
 ```ts file="./SelectMultiTypeaheadCheckbox.tsx"
 
 ```
 
-### View more
+### With view more 
+
+To reduce the processing load for long select lists, replace overflow items with a "View more" link at the bottom of the select menu.
+
+You can adjust the number of items shown above the "View more" link as needed. The following example passes 3 items into this property.
```

Comment:
Wondering if we should mention that the consumer needs to manually create a `<SelectOption>` that acts as a "view more" action, like "To reduce the processing load for long select lists, you can create a `<SelectOption>` as a "view more" action that will render additional options." Depends whether you think we need to outright mention the use of the SelectOption, though; if not then what you have here is good.

Reply:
I don't think we need to get too verbose in the implementation details but I do think calling out something like "use a "View more" link to load additional chunks of select options" would be helpful.

The last sentence is a little confusing to be because it refers to the example implementation rather than a select property. Maybe something like "The following example shows 3 items before the "View more" link.", or combine with the previous sentence?

Reply:
I don't think we need to go too much into the implementation details, but a callout that the view more link serves to load additional chunks of select options would be helpful to have (over just referring to an overflow).

The last sentence is a little confusing because it's referencing the example implementation rather than a Select property, so I think it could be rephrased or combined with the previous sentence like "The following example shows 3 options before the "View more" link."